docs/integrations/skopt.rst

```diff
@@ -1,48 +1,103 @@
+- save the pickled results object.
+
+.. tip::
+    You can log many other experiment metadata like interactive charts, video, audio, and more.
+    See the :ref:`full list <what-you-can-log>`.
+	
+.. note::
+
+    This integration is tested with ``neptune-client==0.4.130``, ``neptune-contrib==0.25.0``, and ``scikit-optimize==0.8.1``
+	
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`quickstart <skopt-quickstart>` below. 
+You can also skip the basics and take a look at how to change what you want to log after training in the :ref:`advanced options <skopt-advanced-options>` section.
+
+If you want to try things out and focus only on the code you can either:
+
+#. Open the Colab notebook (badge-link below) with quickstart code and run it as an anonymous user "`neptuner`" - zero setup, it just works,
+#. View quickstart code as a plain Python script on GitHub.
+
+|Run on Colab|
+
+.. _skopt-quickstart:
 
-    pip install neptune-client neptune-contrib['monitoring']
+Quickstart
+----------
+This quickstart will show you how to:
 
+* Install the necessary neptune packages
+* Connect Neptune to your Skopt hyperparameter tuning code and create the first experiment
+* Log metrics, figures, and artifacts from your tuning job to Neptune, and 
+* Explore them in the Neptune UI.
 
-Initialize Neptune and create an experiment
--------------------------------------------
+.. _skopt-before-you-start-basic:
 
+Before you start
+^^^^^^^^^^^^^^^^
+You have ``Python 3.x`` and following libraries installed:
+
+* ``neptune-client``, and ``neptune-contrib``. See :ref:`neptune-client installation guide <installation-neptune-client>`.
+* ``scikit-optimize==0.8.1``. See |skopt-install|.
+
+You also need minimal familiarity with skopt. Have a look at the |skopt-guide| guide to get started.
+
+.. code-block:: bash
+	
+   pip install --quiet scikit-optimize==0.8.1 neptune-client neptune-contrib['monitoring']
+
+Step 1: Initialize Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. code-block:: python3
 
     import neptune
 
     neptune.init(api_token='ANONYMOUS',
-                 project_qualified_name='shared/showroom')
-    neptune.create_experiment(name='skopt sweep')
+                 project_qualified_name='shared/scikit-optimize-integration')
+				 
+.. tip::
+
+    You can also use your personal API token. Read more about how to :ref:`securely set the Neptune API token <how-to-setup-api-token>`.
+	
+Step 2: Create an Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: python3
 
+    neptune.create_experiment(name='skopt sweep')
 
-Create **NeptuneCallback**
---------------------------
-Pass the experiment object as the first argument.
+This also creates a link to the experiment. Open the link in a new tab. 
+The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
 
+Step 3: Create the Neptune Callback
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'd add a simple text here but use the :meth: .rst reference to link to the NeptuneCallback function in the API reference.